test/xml/unit_plot_data_plane.xml

```diff
@@ -343,6 +343,7 @@
     </output>
   </test>
 
+  <!-- The delta of lat/lon is not con
```

Comment:
Rather than just commenting out these tests, why don't we just remove them entirely?
Is there a reason they should be left commented out?

Reply:
Deleted test cases which are commented out.